devito/symbolics/extended_sympy.py

```diff
@@ -357,6 +357,14 @@ class Macro(sympy.Symbol):
     pass
 
 
+class Literal(sympy.Symbol):
```

Comment:
Why is this needed? Can't you use the `Symbol` class in devito.types?

Reply:
Because of the check in `utils.py` I believe.

Reply:
I get that the check needs an extra, but I don't see why it needs to create a new (mostly copy) of an already existing class. just use `devito.symbol` and same in the check in utils.

Reply:
You can't use Symbol in that check though, it would break using symbols as arguments

Reply:
Symbol works, but I could break other locations. Here I am creating a type for this specific case.